src/joining-the-team.rst

```diff
@@ -0,0 +1,35 @@
+Joining the team!
+*****************
+
+``conda-forge`` is a volunteer-driven community. We need your help! If
+you are interested in helping us maintain this ecosystem, our biggest
+need is help reviewing new conda packages coming in through staged-recipes.
+If you would like to help us do this please ask one of the core members on
+`our gitter channel <https://gitter.im/conda-forge/conda-forge.github.io>`__.
+We will then reach consensus in private and let you know our decision.
+
+Our decision will likely be "yes" but it could be "please get more involved
+with the review process on staged recipes first" if we have not seen you
+take much interest in the staged-recipes review process.
+
+For transparency, our process is to have the core team "vote" on adding new
+members. When we have a new member candidate, we ask the core team to vote
+yes/no and give the team 24h to vote. The person will then be given merge
+rights to conda-forge/staged-recipes so that they can help us review and merge
+submissions more quickly!
+
+staged-recipes responsibilities
+===============================
+You are the welcoming committee for new recipes coming in to the conda-forge
+community! Please give new (and experienced) contributors a pleasant experience!
+Generally speaking your role is as follows:
+
+1. Keep up to date with the current best practices for conda packaging standards
+2. Provide recipe review which generally means making sure that the recipe
+   under review adheres to what we list on the :ref:`recipe`__ page.
+3. <Other people please chime in here with what the responsibilities are supposed to be>
```

Comment:
Open issues as needed (both on staged and on the other flagship repos (smithy, webservices, etc)), especially when problems occur.

Reply:
@cj-wright can you push a commit with the suggested changes?